pycon/finaid/views.py

```diff
@@ -425,3 +425,33 @@ def get_value(name, object):
         writer.writerow(data)
 
     return response
+
+
+@login_required
+def receipt_form(request):
+    receipts = Receipt.objects.filter(user__username=request.user.username)
```

Comment:
You can filter on receipts with a particular user by referring to the user field directly; no need to look at the username.  E.g. `Receipt.objects.filter(user=request.user)`.

Or even slightly shorter, you could write `request.user.financial_aid_receipts.all()`.